bitsail-base/src/main/java/com/bytedance/bitsail/base/packages/FileSystemPluginExplorer.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.bytedance.bitsail.base.packages;
+
+import com.bytedance.bitsail.base.execution.ExecutionEnviron;
+import com.bytedance.bitsail.common.BitSailException;
+import com.bytedance.bitsail.common.configuration.BitSailConfiguration;
+import com.bytedance.bitsail.common.option.CommonOptions;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.collections.CollectionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.net.URI;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileSystemPluginExplorer implements PluginExplorer {
+  private static final Logger LOG = LoggerFactory.getLogger(FileSystemPluginExplorer.class);
+
+  private static final String DEFAULT_PLUGIN_EXPLORER_NAME = "filesystem";
+
+  private ExecutionEnviron execution;
+  private PluginStore pluginStore;
+  private URLClassLoader pluginClassloader;
+
+  @Override
+  public void configure(ExecutionEnviron execution, BitSailConfiguration commonConfiguration) {
+    this.execution = execution;
+
+    String frameworkBaseDir = commonConfiguration
+        .getUnNecessaryOption(CommonOptions.JOB_PLUGIN_ROOT_PATH, getFrameworkEntryDir().toString());
+
+    String pluginDirName = commonConfiguration.get(CommonOptions.JOB_PLUGIN_LIB_PATH);
```

Comment:
The conf `CommonOptions.JOB_PLUGIN_LIB_PATH` only contains the plugin directory name or the plugin full path? If only contains directory name, we'd better change to change the name to `JOB_PLUGIN_DIR_NAME`.